lib/driver.js

```diff
@@ -1,36 +1,91 @@
-import { BaseDriver } from 'appium/driver';
-import log from './logger';
-import * as commands from './commands';
+import {BaseDriver} from 'appium/driver';
 import LRU from 'lru-cache';
+import {
+  activateApp,
+  click,
+  executeRoku,
+  findElOrEls,
+  focus,
+  focusElement,
+  getPageSource,
+  getScreenshot,
+  installApp,
+  makeNextFocusMove,
+  performActions,
+  performActionWithKeyboard,
+  removeApp,
+  rokuEcp,
+  roku_activateApp,
+  roku_appUI,
+  roku_deviceInfo,
+  roku_getApps,
+  roku_installApp,
+  roku_pressKey,
+  setValue,
+  setValueByEcp,
+  setValueByKeyboard,
+  _getElementFromNode,
+  _getXPathNodes,
+} from './commands';
```

Comment:
so yeah, this is gnarly, but it is easier than doing the mixin dance.

Reply:
yeah :-( i do like it better than the weird mixin chain strategy